website/index.html

```diff
@@ -29,9 +49,6 @@
     <div id="root">
       <div class="spinner"></div>
     </div>
-    <!--/placeholder1-->
-    <script src="/src/index.tsx" async type="module"></script>
-    <!--placeholder2-->
-    <!--/placeholder2-->
```

Comment:
Those placeholder tags (both open and close) are required now.

Reply:
I added them again in the latest commit. I believe it is correct now.